### DISCUSSION TITLE
/albumart/ not working after update from 9.1.1 -> 9.2.1

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#709] [Q&A] [answered]
myMPD version: 9.2.1

updated from version 9.1.1 -> 9.2.1 and after that, api-call for getting albumart is not longer possible:

`
Embedded asset "/albumart/alben/..." not found.`

The path seem to be exported correctly.

**Comment by jcorporation** [comment, marked as answer]
With 9.2.0 the uri has changed: https://jcorporation.github.io/myMPD/references/webserver-uris

**Comment by payback007** [comment]
how has it to be coded, to get the correct webserver path for display the current cover? I'm not really familiar wth such calls.
Up to now I used this code:


```
#!/bin/sh

URI=$(curl -s -k http://IP:PORT/api/ -d'{"jsonrpc":"2.0","id":0,"method":"MYMPD_API_PLAYER_CURRENT_SONG","params":{}}' | jq -r ".result.$
echo "http://IP:PORT/albumart/${URI}"
```

**Comment by jcorporation** [comment]
This is the correct call to get the cover for the uri:

`echo "http://IP:PORT/albumart?offset=0&uri=${URI}"`

**Comment by payback007** [comment]
Thank you very much, now working for all of the album arts, but it's not working of a manually added cover for radio stream. I'm not using the new webradio feature, I set a stream in MPD and the cover is correcty shown in mympd, but some browsers seem to have a problem to correctly display cover of the stream. Instead the radio cover, the export picture showing the "green radio icon" from mympd. 

Any idea about this? In the previous version it's shown correctly.

**Reply by jcorporation** [reply]
Please provide debug logs for the described issue.
